rasa/nlu/run.py

```diff
@@ -20,7 +20,7 @@ def run_cmdline(model_path, component_builder=None):
 
 if __name__ == "__main__":
     raise RuntimeError(
-        "Calling `rasa.nlu.server` directly is "
-        "no longer supported. "
-        "Please use `rasa shell` instead."
+        "Calling `rasa.nlu.run` directly is no longer supported. "
+        "Please use `rasa run` to start a Rasa server or `rasa shell` to obtain "
+        "the intent and entities of some text via the command line."
```

Comment:
Not sure if you have seen my latest comment on the CLI issue 

>When using rasa run, if the model located in the default directory is an NLU model and not a full stack model, the command line becomes an NLU interpreter

 `rasa run nlu` should have the above functionality if the model is either an NLU model or a stack model. Currently it runs the bot instead of running the nlu interpreter, even if the model is only an NLU model.

I am not sure if `rasa shell` actually acts as an NLU interpreter in this case -- it was `rasa run` that was working as an interpreter for tom for an NLU model (though now it doesn't work)

Reply:
I would do it like this:
- `rasa run` starts a server. It does not matter what kind of model is provided. The server can run with all kind of models.
- `rasa shell` runs the bot on the command line, if the model includes a Core model. If just an NLU model is given, an NLU interpreter is started.

We can also make it more explicit: Add a subparser for `rasa shell nlu` that always starts an NLU interpreter as long as an NLU model is included in the provided model.

Reply:
Ah okay, so if you have an NLU model, `rasa run` runs an NLU server? If so, then it'd be good if `rasa run nlu` just ran an NLU server for a stack model, same with core (this may already be the case, not sure).

The way you've explained it I think it does make sense to put the NLU interpreter functionality in `shell` instead of `run` since its not actually launching a server -- i like the idea of a subparser too, it'd be really helpful to be able to test out the NLU for a stacked model too

Reply:
Will add it to the ticket.

Reply:
Awesome, just hit me with another review when you're ready!

Reply:
Sorry for the misunderstanding, will do it in another PR.